samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -289,13 +289,22 @@ object SamzaContainer extends Logging {
 
     info("Got change log system streams: %s" format changeLogSystemStreams)
 
+    val accessLogSystemStreams = config
+      .getStoreNames
+      .filter(config.getAccessLogStream(_).isDefined)
+      .map(name => (name, config.getAccessLogStream(name).get)).toMap
+      .mapValues(Util.getSystemStreamFromNames(_))
+
+    info("Got access log system streams: %s " format accessLogSystemStreams)
+
     val serdeManager = new SerdeManager(
       serdes = serdes,
       systemKeySerdes = systemKeySerdes,
       systemMessageSerdes = systemMessageSerdes,
       systemStreamKeySerdes = systemStreamKeySerdes,
       systemStreamMessageSerdes = systemStreamMessageSerdes,
-      changeLogSystemStreams = changeLogSystemStreams.values.toSet)
+      changeLogSystemStreams = changeLogSystemStreams.values.toSet,
+      accessLogSystemStreams = accessLogSystemStreams.values.toSet)
```

Comment:
As discussed with @prateekm, I have now removed the extra parameter from the `StorageEngineFactory` interface